gnosis/cowswap
# Remove unsupported list

# Summary

This PR removes the list of unsupported tokens inherited from Uniswap. The backend is the one telling the Dapp about the us-upported tokens, so an empty list is used instead.

Left the list, just to keep the change minimal and allow to add tokens in the future if we need to. Also, uses our own image for the logo (256x258 18 color image) Thanks @michelbio from the tips on how to generate and optimize!



# To Test

1. Try to see if you can swap any token in https://github.com/gnosis/cowswap/blob/6fb3ded2ba3bbc284477b1c2d78bf2a4bf201538/src/constants/tokenLists/unsupported.tokenlist.json#L14

NOTE: Images might not load.
